src/runtime/service_api_c.cpp

```diff
@@ -523,7 +535,7 @@ bool run(const char *config_file,
     // start the tool
     dsn_all.tool->run();
 
-    if (sleep_after_init) {
+    if (is_server) {
```

Comment:
Why server should sleep? and when `is_server = true` and when `is_server=false`? besides, do we need `sleep_after_init` any more?

Reply:
If it is a server, the startup thread should sleep. In previous implementation, it use `sleep_after_init` instead of `is_server`. So I change it to `is_server` in this pull request